packages/ngtools/webpack/src/transformers/ast_helpers.ts

```diff
@@ -50,25 +50,26 @@ export function getLastNode(sourceFile: ts.SourceFile): ts.Node | null {
 // Test transform helpers.
 const basePath = '/project/src/';
 const fileName = basePath + 'test-file.ts';
-const tsLibFiles = loadTsLibFiles();
+const tsLibFiles = loadTypeScriptLibFiles();
 
 export function createTypescriptContext(
   content: string,
   additionalFiles?: Record<string, string>,
   useLibs = false,
-  importHelpers = true,
+  extraCompilerOptions: ts.CompilerOptions = {},
 ) {
   // Set compiler options.
   const compilerOptions: ts.CompilerOptions = {
-    noEmitOnError: useLibs,
+    noEmitOnError: false,
```

Comment:
Discussed offline, setting this to false means our transforms can be generating code that fails TS compilation, and we wouldn't know from running these tests. If there's some way we can not set this option at all it would be great.

Reply:
Should be fixed now. Note: we do have quite a number of tests that use `noEmitOnError: false` because of imports to `@angular/core` etc...

Reply:
Thank you, we should eventually get to review that... but at least we're not introducing more now.